pkg/package_manager.go

```diff
@@ -35,6 +43,7 @@ func NewPackageManager(res *Instance) *PackageManager {
 
 	return &PackageManager{
 		instance: res,
+		config:   res.manager.aem.baseOpts,
```

Comment:
this field is not needed; in a place you are accessing it use `instance` field directly, or introduce shorthand private method